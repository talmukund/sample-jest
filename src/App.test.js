
import React from 'react';
import App from './App';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

it('renders correctly', () => {
    const wrapper = shallow(<App />)
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
});
