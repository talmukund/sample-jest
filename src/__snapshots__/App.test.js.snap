// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header>
    <h1>
      Welcome to React
    </h1>
  </header>
  <p>
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
